quadrat/sass/_utility.scss

```diff
@@ -0,0 +1,7 @@
+.image-no-margin {
```

Comment:
Nice candidate to move to BCB eventually?

Reply:
Yes, I think so, I found out there's a .com plugin that does exactly this but adding a toggle inside the editor!